src/components/Navbar.jsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom";
+
+const Navbar = () => {
+  return (
+    <nav className="navbar">
+      <div className="links">
+        <Link to="/">Home</Link>
+        <Link>Coding</Link>
+        <Link>Football</Link>
+        <Link>Cooking</Link>
```

Comment:
I'd make sure you change these from hardcoded values when you get upto it - you've an api route that will give you all the topics from the db - that way they're always upto date if more get added

Reply:
will do I just wanted to fill the navbar with initial data :)